packages/checkout/sdk/src/smartCheckout/routing/bridge/getBridgeFeeEstimate.ts

```diff
@@ -24,7 +24,7 @@ export const getBridgeFeeEstimate = async (
     const bridgeFeeResponse = await bridge.getFee(
       {
         action: BridgeFeeActions.DEPOSIT,
-        gasMultiplier: 1.1,
+        gasMultiplier: 'auto',
```

Comment:
Nit: Any reason we do not use a `const` for this value?

Reply:
https://immutable.atlassian.net/browse/SMR-2885